src/SignalR/common/Protocols.Json/src/JsonHubProtocolOptions.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.AspNetCore.SignalR
 {
     /// <summary>
-    /// Options used to configure a <see cref="JsonHubProtocolOptions"/> instance.
+    /// Options used to configure a <see cref="JsonSerializerOptions"/> instance.
```

Comment:
```suggestion
    /// Options used to configure a <see cref="JsonHubProtocol"/> instance.
```

Reply:
Or would the following be better?

```suggestion
    /// Options used to configure the <see cref="JsonSerializerOptions"/> used by SignalR's <see cref="JsonHubProtocol"/> instance.
```

Reply:
That's ok, my main concern is that these options can be for more than just the Serializer, like maybe we'll allow different protocol versions via these options in the future.